blocks/init/src/Blocks/custom/lottie/assets/lottie-control.js

```diff
@@ -0,0 +1,232 @@
+import lottie from 'lottie-web';
+import { cleanObject } from '../../../assets/scripts/helpers/cleanObject';
```

Comment:
load it from the new location using @frontend-libs alias like in the other examples as this will not work this way when moved to your project